inc/Engine/HealthCheck/CacheDirSizeCheck.php

```diff
@@ -89,8 +88,7 @@ public function cache_dir_size_check() {
 		$current_blog_id = get_current_blog_id();
 
 		$checks = [
-			'min'     => WP_ROCKET_MINIFY_CACHE_PATH . $current_blog_id,
-			'busting' => WP_ROCKET_CACHE_BUSTING_PATH . $current_blog_id,
+			'min' => rocket_get_constant( 'WP_ROCKET_MINIFY_CACHE_PATH' ) . $current_blog_id,
```

Comment:
Let's move the minify cache path to be a property instead that's set in the constructor. Why? If `cache_dir_size_check()` method is called more than once, the code will run a tiny bit faster as the minify path doesn't have to build again and again.

Reply:
https://github.com/wp-media/wp-rocket/pull/2616/commits/9e95cf8cd5eb809236b17004a12c3b43715b4c56